src/golang/lib/workflow/operator/service.go

```diff
@@ -6,13 +6,16 @@ import (
 
 	"github.com/aqueducthq/aqueduct/lib/database"
 	"github.com/aqueducthq/aqueduct/lib/dynamic"
+	"github.com/aqueducthq/aqueduct/lib/functional/slices"
+	"github.com/aqueducthq/aqueduct/lib/lib_utils"
 	"github.com/aqueducthq/aqueduct/lib/models"
 	"github.com/aqueducthq/aqueduct/lib/models/shared"
 	"github.com/aqueducthq/aqueduct/lib/repos"
 	"github.com/dropbox/godropbox/errors"
 	"github.com/google/uuid"
 )
 
+// GetOperatorsOnIntegraiton will return an empty list for notification resources.
```

Comment:
@likawind so instead of fetching the operators for notification resources, which would lead to unnecessary performance cost, I just had callers that cared about notifications use `GetWorkflowIDsUsingNotification()` instead. Callers that don't care about returning no operators for notifications will just get an empty list. This meant a couple extra checks in the frontend, but it seemed to be the fastest way forward without incurring too much debt.